pandas/core/series.py

```diff
@@ -1814,7 +1814,7 @@ def combine_first(self, other):
         this = self.reindex(new_index, copy=False)
         other = other.reindex(new_index, copy=False)
         # TODO: do we need name?
-        name = com._maybe_match_name(self, other)  # noqa
```

Comment:
anyidea why the noqa is here?

Reply:
I'd speculate that it's because `name` is defined and then for some reason not used, so flake8 _should_ complain.